### PR TITLE
Flux cleanup

### PR DESCRIPTION
**Pull request by jameshcorbett**
Cleaning up some of the Flux code.

### CODE REVIEW

File: scripts/python/scrjob/launchers/flux.py [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
 from scrjob.common import runproc
 from scrjob.launchers import JobLauncher
 
 
 class FLUX(JobLauncher):
+    """Handles launching and monitoring Flux jobs."""
 
     def __init__(self, launcher='flux'):
-        super(FLUX, self).__init__(launcher=launcher)
+        super().__init__(launcher=launcher)
```
Comment (mcfadden8) [inline scripts/python/scrjob/launchers/flux.py:17]: This change was for Python 3.0?  I'm curious, did this line work before (I'm fine with this change btw, I'm just curious)

File: scripts/python/scrjob/launchers/flux.py [hunk 2]
```diff
@@ -43,19 +46,19 @@ def wait_run(self, proc, timeout=None):
...
             return False, None
-        except Exception as e:
+        except Exception as exc:
             # it can also throw an exception if there is no job to wait for
-            print(e)
+            print(exc)
             return False, None
 
-        if success == False:
+        if success is False:
```
Comment (mcfadden8) [inline scripts/python/scrjob/launchers/flux.py:61]: Was this line previously broken?  I assume it was?

**Review by mcfadden8 — COMMENTED** [review]
Looks great, thanks!  I just had a couple of questions inline.  I'm new here, but do we have CI that tests this?  I imagine this may be difficult or not possible?

File: scripts/python/scrjob/launchers/flux.py (hunk 1 above)
Comment (jameshcorbett) [reply scripts/python/scrjob/launchers/flux.py:17]: Yeah it did work before, it's just superfluous in Python 3.

File: scripts/python/scrjob/launchers/flux.py (hunk 2 above)
Comment (jameshcorbett) [reply scripts/python/scrjob/launchers/flux.py:61]: Not broken, just not idiomatic Python.

**Comment by jameshcorbett** [comment]
> Looks great, thanks! I just had a couple of questions inline. I'm new here, but do we have CI that tests this? I imagine this may be difficult or not possible?

We should definitely get CI that tests this if we don't have it!

**Comment by mcfadden8** [comment]
> > Looks great, thanks! I just had a couple of questions inline. I'm new here, but do we have CI that tests this? I imagine this may be difficult or not possible?
> 
> We should definitely get CI that tests this if we don't have it!

Thanks @jameshcorbett.  @gonsie, do you know whether our CI tests this?  If not, we can at least write up an issue to get it covered at some point.

**Review by mcfadden8 — APPROVED** [review]

**Pull request merged**